docs/boshdirector.adoc

```diff
@@ -28,114 +40,307 @@ Click on the zone; you'll notice an NS-type entry for the subdomain, as shown be
+|Checked
+
+|===
 
 === Create Availability Zones
 
-See http://docs.pivotal.io/pivotalcf/1-9/customizing/gcp-om-config.html#az[instructions section^].
+. Review http://docs.pivotal.io/pivotalcf/1-9/customizing/gcp-om-config.html#az[availability zones^].  Browse to the `Create Availability Zones` section (left margin).
+
++
+[.thumb]
+.Availability Zones
+image::create-availability-zones-gcp.png[Availability Zones]
+
 
-Create three availability zones (e.g. us-central1-a, us-central1-b, us-central1-c).
+. Add three availability zones based on your region,  These will be the same zones you put into your `terraform.tfvars` terraform script.  Click `Save` after entering each zone.
 
++
 [.thumb]
 .Availability Zone Configuration
-image::az_config.png[Availability Zone Configiuration]
-
-Press _Save_.
+image::az_config.png[Availability Zone Configuration]
 
++
+**Question** According to the documentation of AZs, how many availability zones _should_ you create? Why?
 
 === Create Networks
 
-See http://docs.pivotal.io/pivotalcf/1-9/customizing/gcp-om-config.html#network[instructions section^].
+. Review http://docs.pivotal.io/pivotalcf/1-9/customizing/gcp-om-config.html#network[networks^].  Browse to the `Create Networks` section (left margin).  Pay particular attention to the _Google Network Name_ format.  You can obtain the network subnets from the GCP web console:  select Networks -> Network, and look for your `{env_name}-pcf-network`.  You should see 3 subnetworks created: `{env_name}-ert-subnet`, `{env_name}-management-subnet`, and `{env_name}-services-subnet`.
+
++
+[.thumb]
+.Create Networks
+image::create-networks-gcp.png[Create Networks]
 
-Create three networks to match the three subnets that were created by the terraform scripts.  Here's an example configuration for the elastic runtime network:
+. Add three networks to match the three subnets that were created by the terraform scripts.
 
++
+.Management Network
+|===
+|Field|Value
+
+|Name
+|gcp-management-network
+
+|Google Network Name
+|{env_name}-pcf-network/{env_name}-management-subnet/{region}
+
+|CIDR
+|10.0.0.0/24
+
+|Reserved IP Ranges
+|10.0.0.1-10.0.0.9
+
+|DNS
+|10.0.0.1
+
+|Gateway
+|10.0.0.1
+
+|Availability Zones
+|Check all zones
+
+|===
+
++
+.Elastic Runtime (ERT) Network
+|===
+|Field|Value
+
+|Name
+|gcp-ert-network
+
+|Google Network Name
+|{env_name}-pcf-network/{env_name}-ert-subnet/{region}
+
+|CIDR
+|10.0.4.0/22
+
+|Reserved IP Ranges
+|10.0.4.1-10.0.4.9
+
+|DNS
+|10.0.4.1
+
+|Gateway
+|10.0.4.1
+
+|Availability Zones
+|Check all zones
+
+|===
+
++
+.Service Network
+|===
+|Field|Value
+
+|Name
+|gcp-services-network
+
+|Service Network
+|Checked
+
+|Google Network Name
+|{env_name}-pcf-network/{env_name}-services-subnet/{region}
+
+|CIDR
+|10.0.8.0/22
+
+|Reserved IP Ranges
+|10.0.8.1-10.0.8.9
+
+|DNS
+|10.0.8.1
+
+|Gateway
+|10.0.8.1
+
+|Availability Zones
+|Check all zones
+
+|===
+
++
+Here's an example configuration for the elastic runtime network:
+
++
```

Comment:
Given the time constraints the class is under, we really don't have time for students to make a lot of configuration errors in the class.  So I'd rather err on the side of too much handholding over letting them figure it out.  Brighter students tend to push ahead without reading the docs - but a screenshot is a great visual spot check they can use.